constants/constants.go

```diff
@@ -0,0 +1,16 @@
+package constants
+
+const (
+	TaxRateForRAW                                    = 12.50
```

Comment:
RAWItemTaxRate ?

Reply:
okay! added proper constant and variable names